src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Requests/ProviderPermissions/GetProviderAccountLegalEntitiesRequest.cs

```diff
@@ -1,15 +1,26 @@
-﻿using SFA.DAS.SharedOuterApi.Interfaces;
+﻿using System.Collections.Generic;
+using Microsoft.IdentityModel.Tokens;
+using SFA.DAS.SharedOuterApi.Interfaces;
+using SFA.DAS.SharedOuterApi.Models;
 
 namespace SFA.DAS.SharedOuterApi.InnerApi.Requests;
 
-public class GetProviderAccountLegalEntitiesRequest : IGetApiRequest
+public class GetProviderAccountLegalEntitiesRequest : IGetApiRequest﻿
 {
     private readonly int? _ukprn;
+    private readonly string _operations = "";
 
-    public GetProviderAccountLegalEntitiesRequest(int? ukprn)
+    public GetProviderAccountLegalEntitiesRequest(int? ukprn, List<Operation> operations)
     {
         _ukprn = ukprn;
+        if (!operations.IsNullOrEmpty())
+        {
+            foreach (Operation operation in operations)
+                this._operations += $"&operations={(int)operation}";
+        }
+        else
+            _operations = "&operations=";
     }
```

Comment:
Also by defaulting operations parameter, you could have avoided all the failures in other areas. Thinking about it there was no need to add the operations to the request at all as it would have worked with out it, but I glad you did, we can use it elsewhere.